state-network.md

```diff
@@ -255,53 +269,297 @@ sedes      := Container(enrs: List[byte_list, max_length=32], payload: byte_list
 > A response with an empty `payload` and empty `enrs` indicates that the node is not aware of any closer nodes, *nor* does the node have the requested content.
 
 
-#### Advertise (0x07)
+#### Offer (0x07)
 
-Advertise a set of content keys that this node has proofs available for.
+Offer a set of content keys that this node has proofs available for.
 
 ```
 message_id := 7
 type       := request
-sedes      := List[byte_list, max_length=32]
+sedes      := List[byte_list, max_length=64]
```

Comment:
In chain history for `Offer` we are using `Container(content-keys: List[Bytes, 32])`, not bare list. It would be good to make it consistent imo. ( I slightly prefer version with Container)

Reply:
Indeed. There are other locations such as `Find Content (0x05)` where there is only 1 item yet it gets wrapped in a container. So either we get Container on everything, or not at all on single items, but not some mix. I prefer the first.

Reply:
:+1: I'll switch to container.  I like the consistency too